packages/odyssey-react-mui/src/labs/SideNav.tsx

```diff
@@ -577,16 +589,43 @@ const SideNav = ({
 }: SideNavProps) => {
   const [isSideNavCollapsed, setSideNavCollapsed] = useState(false);
   const odysseyDesignTokens = useOdysseyDesignTokens();
+  const [sideNavItemIdToScrollTo, setSideNavItemIdToScrollTo] =
+    useState<string>("");
+  const scrollToNode = useCallback<RefCallback<HTMLLIElement>>((node) => {
+    node?.scrollIntoView({
+      behavior: "instant",
+      block: "end",
+      inline: "nearest",
+    });
+  }, []);
+  const scrollToNodeRefIfApplicable = useCallback(
+    (itemId: string) =>
```

Comment:
I know you know this change would also need to be made but, just flagging just in case. If we should choose the more verbose route mentioned below
```js
(itemId: string, element: HTMLLIElement | null) => ...
```